src/common/components/data-display/text/SectionTitle.tsx

```diff
@@ -19,25 +13,20 @@ const useStyles = makeStyles(() => {
       display: "flex",
       alignItems: "center",
     },
+    sectionTitle: {
+      color: "#f2f2f2",
```

Comment:
Maybe would make sense to define the text colors in theme? Would be great not to define it in every component that uses this text color, and there seem to be many.